client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -156,13 +161,11 @@ object CapiModelEnrichment {
 
       // Special Report hashes can be generated by executing:
       // echo -n '<salt><tag-id>' | md5sum
-
       val hashedSpecialReportTags: Set[String] = Set(
         "0d18e8413ab7cdf377e1202d24452e63"
```

Comment:
I think this one is `news/series/uber-files` tag. Perhaps in a follow-up PR you could "reveal" this as well?

Reply:
Good idea and thanks for working that out. I'll do that.